bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/editor/BlueprintEditor.scala

```diff
@@ -31,6 +31,7 @@ object BlueprintEditor {
       case e:ExchangeProposalClass => e.partner match {
         case ExchangePartner.KECK   => Site.Keck
         case ExchangePartner.SUBARU => Site.Subaru
+        case ExchangePartner.KECK   => Site.CFHT // Shouldn't happen
```

Comment:
Good catch, this is not logically reachable code but it should be correct in any case